learn/guides/developing-a-restful-api-with-ballerina.md

```diff
@@ -0,0 +1,249 @@
+---
+layout: developing-a-restful-api-with-ballerina
+title: Developing a RESTful API with Ballerina
+permalink: /learn/developing-a-restful-api-with-ballerina/
+description: This simple guide helps you understand the basics of Ballerina constructs which allow you to write RESTful APIs. 
+active: developing-a-restful-api-with-ballerina
+intro: This simple guide helps you understand the basics of Ballerina constructs which allow you to write RESTful APIs. 
+redirect_from:
+  - /learn/developing-a-restful-api-with-ballerina
+---
+
+This simple guide helps you understand the basics of Ballerina constructs which allow you to write RESTful APIs. 
+
+Due to the batteries included nature of Ballerin language there is no need to add any third party libraries to implement the RESTful API. The Ballerina standard library itself is adequate. In this API you will be writing a simple CRUD like RESTful service. 
+
+To get the best out of the guide it is better to have some idea about Ballerina.
+
+In this tutorial, you will be creating a RESTful API with two endpoints. The sample is built around a set of covid19 data.
+
+The tutorial includes the following steps,
```

Comment:
```suggestion
The tutorial includes the following steps,
```
Do we need this? The left navigation will show the sub topics.

Reply:
We thought leaving it for now.